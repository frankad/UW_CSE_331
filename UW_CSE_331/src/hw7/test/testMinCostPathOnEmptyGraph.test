# Tests a graph created without nodes or simply an empty graph.

# Load emptyGraph.tsv
LoadGraph emptyGraph emptyGraph.tsv

# Print the list of nodes in the graph, emptyGraph
ListNodes emptyGraph

# find the minimum cost path between some random nodes that are not exist in
# the given empty graph.
FindPath emptyGraph Eva John