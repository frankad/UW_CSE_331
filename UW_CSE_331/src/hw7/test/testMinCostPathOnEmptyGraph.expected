# Tests a graph created without nodes or simply an empty graph.

# Load emptyGraph.tsv
loaded graph emptyGraph

# Print the list of nodes in the graph, emptyGraph
emptyGraph contains:

# find the minimum cost path between some random nodes that are not exist in
# the given empty graph.
unknown character Eva
unknown character John